examples/reflection/generic_reflection.rs

```diff
@@ -13,7 +13,7 @@ fn main() {
 }
 
 #[derive(Reflect)]
-struct MyType<T: Reflect> {
+struct MyType<T: Reflect + TypePath> {
```

Comment:
Nit: Same here. This isn't needed anymore, right?
```suggestion
struct MyType<T: Reflect> {
```

Reply:
Resolved